server/tests/message.integration.spec.js

```diff
@@ -357,57 +374,60 @@ describe('Message API:', function () {
 
     });
     
-    // describe('GET /message/count/:userId', function () {
+    describe('GET /message/count', function () {
 
-    //     let userId;
+        let option1 = 'all';
+        let option2 = 'unread';
 
-    //     before(done => {
-    //         Message.destroy({
-    //             where: {},
-    //             truncate: true
-    //         }).then(() => {
-    //             Message.create(testMessageObject)
-    //                 .then(message => {
-    //                     userId = message.from;
-    //                     done();
-    //                 });
-    //         });
-    //     });
+        // before is called twice to make sure a user has alteast 3 messages sent to him
+        before(() => request(app)

```

Comment:
Because you are using the same code multiple times, make this a single function with the test message object as the parameter.